### PR TITLE
Minor Code Cleanup

### PR DESCRIPTION
**Pull request by marvin-j97**
- Fix broken link in Readme
- Fix typo (getRanomChar)
- Fix typo (Exmaple)
- Use more accurate type for shuffle

### CODE REVIEW

File: dev/src/pages/group.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Component, For, createSignal } from "solid-js";
 import { TransitionGroup } from "solid-transition-group";
 
-function shuffle<T extends any[]>(array: T): T {
+function shuffle<T>(array: T[]): T[] {
```
Comment (thetarnav) [inline dev/src/pages/group.tsx:4]: There is a slight difference between the two. [(example)](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwGcALZRRCEARgB4AVeEADwxFWAPilQE8BtAXQB8AClgwo3AFzxaASmm0AUItCRYCFOmx5CJMhQBMdEWIkKB8mQOWroceGDwEM8VMgC2AIxAwA+mOk3Lx9rW3UHJxdnGCxUAHN-GBhpaNi40PA7BEdUZ1cPbz9U+MTk+GFiuPgAH3zgmFkMtXscvKDC0t9K0pSMGPiBGrrC60VWlzgCZAgMX0o5+ABeXVJyKmF2n1LZRQB6XfhDw4A9AH4xyPhJ6dn5gyWV-XXusR39o5Pz8auQKZm53wAZgexFWFEoGwKWxeSTeBw+ZwuuQmvxuAIALCC9GsIZs-GIun00ts9vCjoikXlrv8DAtlqCngZIfUSe8EV9LtTZrT7vTsYYKkSSq9SR94IjvlzfLTgXywSAmXjCf0EiK2eSOcifn9ub5MXLGcyOgSYQ1ReygA)
This is why I usually stick to the first one.
But for a dome it doesn't really matter

**Review by thetarnav — COMMENTED** [review]
Thanks
*so many spelling mistakes... 🫣*

File: dev/src/pages/group.tsx (hunk 1 above)
Comment (marvin-j97) [reply dev/src/pages/group.tsx:4]: That's true - I'd say T[] is generally more desirable because you can directly access the array element type.

Comment (thetarnav) [reply dev/src/pages/group.tsx:4]: alright, works for me

**Pull request merged**